packages/block-library/src/image/edit.js

```diff
@@ -509,17 +509,19 @@ export class ImageEdit extends Component {
 			</>
 		);
 
-		const AlignmentWrapper = needsAlignmentWrapper ? Block.div : Fragment;
-		const BlockContentWrapper = needsAlignmentWrapper
-			? 'figure'
-			: Block.figure;
 		// Disable reason: Each block can be selected by clicking on it
 		/* eslint-disable jsx-a11y/click-events-have-key-events */
 		return (
 			<>
 				{ controls }
-				<AlignmentWrapper>
-					<BlockContentWrapper className={ classes }>
+				<div
+					className={
+						needsAlignmentWrapper
+							? 'wp-block block-editor-block-list__block'
```

Comment:
Ideally these classes are not needed, and ideally, we provide an alignment wrapper component that the block can wrap around the block.

Reply:
Why these are needed?

Reply:
I guess it's another argument to absorb the alignment in `Block.*` for now.

Reply:
I'm fine with this for now (with a comment) while we continue the work to improve alignments

Reply:
It's needed so the div has the proper width and margins. Yes, we'd need to comment and continue with figuring out how to best proceed with the alignment issue.